packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -706,6 +722,18 @@ contract OPContractsManager is ISemver {
             IDisputeGameFactory dgf = getDisputeGameFactory(gameConfig.systemConfig);
             setDGFImplementation(dgf, gameConfig.disputeGameType, IDisputeGame(address(outputs[i].faultDisputeGame)));
             dgf.setInitBond(gameConfig.disputeGameType, gameConfig.initialBond);
+
+            if (gameConfig.permissioned) {
+                // Emit event for the newly added game type with the old permissioned dispute game
+                emit GameTypeAdded(
```

Comment:
nit, but can we do this event emission in the above `if (gameConfig.permissioned) / else` block to simplify control flow?

Reply:
Mentioned that here: https://github.com/ethereum-optimism/optimism/pull/14196/files#r1953015117

If you think slightly odd event ordering is OK, we can do that.

Reply:
Ah missed that comment

Hmm interesting, good point. I do not feel strongly here, I'm ok with leaving this as-is